Source/Core/DolphinQt/Host.h

```diff
@@ -27,13 +27,15 @@ class Host final : public QObject
   bool GetRenderFocus();
   bool GetRenderFullFocus();
   bool GetRenderFullscreen();
+  bool GetTASInputFullFocus();
```

Comment:
```suggestion
  bool GetTASInputFullFocus() const;
```
It's probably worth making all of these other getters const as well.